tests/constraint/test_TSR.cpp

```diff
@@ -18,15 +18,15 @@ using dart::common::make_unique;
 using DefaultRNG = RNGWrapper<std::default_random_engine>;
 
 
-static Eigen::Vector6d getPose(Eigen::Isometry3d isometry)
+static inline Eigen::Vector6d getPose(Eigen::Isometry3d isometry)
```

Comment:
It seems the real problem is this function is _unused_. I think we could simply remove this function since it was defined only for the local tests but not used.